crates/pallets/starknet/src/tests/invoke_tx.rs

```diff
@@ -203,6 +203,76 @@ fn given_hardcoded_contract_run_invoke_tx_then_event_is_emitted() {
     });
 }
 
+#[test]
+fn given_hardcoded_contract_run_invoke_tx_then_multiple_events_is_emitted() {
+    new_test_ext().execute_with(|| {
+        System::set_block_number(0);
+        run_to_block(2);
+
+        let emit_contract_address =
+            Felt252Wrapper::from_hex_be("0x051a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02cf").unwrap();
+
+        let sender_account = get_account_address(AccountType::NoValidate);
+
+        let events = System::events();
+
+        let do_one_event_transaction = InvokeTransaction {
+            version: 1,
+            sender_address: sender_account,
+            calldata: bounded_vec![
+                emit_contract_address, // Token address
+                Felt252Wrapper::from_hex_be("0x1f9630afc69718ef80a0353d14ae7e8e4003b0f4def439cfdde1843fdff1833")
```

Comment:
why don't you use `get_selector_from_name` as in the `call_contract` example?

Reply:
i believe they return different types